packages/mailchimp/test/util.test.js

```diff
@@ -0,0 +1,44 @@
+import { expect } from 'chai';
+import { MockAgent, setGlobalDispatcher } from 'undici';
+
+import { request } from '../src/Utils.js';
+
+const mockAgent = new MockAgent({ connections: 1 });
+setGlobalDispatcher(mockAgent);
```

Comment:
I suspect this doesn't work because the client doesn't use the global dispatcher.

Reply:
Yeah your are right, i have removed this